### DISCUSSION TITLE
Is Forking or cloning better in Github

### DISCUSSION THREAD

**Discussion by karan10299201** [karan10299201/demoo#2] [Q&A] [answered]
I cannot find any difference between these two . Can somebody help me differentiate between them

**Comment by pratheekv39** [comment, marked as answer]
Forking is better for contributing to or creating derivatives of existing projects on GitHub while maintaining a link to the original. Cloning is preferable for creating local copies of repositories to work on them locally, ideal for individual or team projects where local development is needed. The choice between forking and cloning depends on whether you aim to contribute to a project or work on it locally.
